components/layout_thread_2020/lib.rs

```diff
@@ -607,10 +607,6 @@ impl LayoutThread {
     }
 
     /// The high-level routine that performs layout.
-    #[cfg_attr(
-        feature = "tracing",
-        tracing::instrument(skip_all, fields(servo_profiling = true), level = "trace")
-    )]
```

Comment:
I think maybe this was removed by accident? By the way, you can fix the Rust analyzer errors by building Servo with tracing enabled. This is what I do in my Visual Studio Code settings:

```json
{
    "rust-analyzer.checkOnSave.overrideCommand": [
        "./mach",
        "clippy",
        "--message-format=json",
        "--target-dir",
        "target/lsp",
        "--features", "layout_2013,tracing-perfetto"
    ],
    "rust-analyzer.cargo.buildScripts.overrideCommand": [
        "./mach",
        "clippy",
        "--message-format=json",
        "--target-dir",
        "target/lsp",
        "--features", "layout_2013,tracing-perfetto"
    ],
    "rust-analyzer.rustfmt.overrideCommand": [ "./mach", "fmt" ],
}
```

Reply:
Yeah I didn't mean to commit this. Thanks!